lib/dolly/document.rb

```diff
@@ -128,7 +128,8 @@ def write_property name, value
     end
 
     def read_property name
-      self.properties[name].value
+      return self.properties[name].value unless doc[name.to_s]
```

Comment:
I would agree.  I guess this helps open up a discussion as to how best to tackle this.  Do you think I should take a look at the property class and see if I hide the logic in there?